src/utils/utils.js

```diff
@@ -273,8 +273,11 @@ var utils = {
     clone(obj) {
         return deepClone(obj);
     },
-    isArray(obj) {
-        return Array.isArray(obj);
+    isDate(obj) {
+        return obj instanceof Date && !isNaN(Number(obj));
```

Comment:
To handle invalid dates which undescore ignore.

_.isDate(new Date('derp')) results in true. However that's not a valid date (converts to NaN). 

Is there any reason to consider invalid dates as dates? They might break other code easily when trying to format it or extract some info from it (which always returns NaN).